demos/Grokking_Demo.ipynb

```diff
@@ -188,7 +190,8 @@
    "outputs": [],
    "source": [
     "# where we save the model\n",
-    "PTH_LOCATION = \"/workspace/_scratch/grokking_demo.pth\""
+    "PTH_LOCATION = \"/workspace/_scratch/grokking_demo.pth\"\n",
+    "assert os.path.exists(Path(PTH_LOCATION).parent), f\"Please create the folder {Path(PTH_LOCATION).parent} before running this notebook\""
```

Comment:
TBH, the `AssertionError` is not really that much fun to see after returning to the Colab after letting it install and run for a few minutes ... would `os.mkdir(...)` be possible instead?

tested in https://colab.research.google.com/github/neelnanda-io/TransformerLens/blob/3f331759ddab8da81df41b55a49b9d79e80b59e6/demos/Grokking_Demo.ipynb

Reply:
also, what's the deal with absolute paths? does Google Colab map root folder to something I can create? relative path (without starting `/`) seems to work fine in Colab (...still training):

<img width="914" alt="Screenshot 2023-11-21 at 17 08 25" src="https://github.com/neelnanda-io/TransformerLens/assets/1087670/c21ac323-a361-40c1-bde8-27d896114916">